zilliqa/src/transaction.rs

```diff
@@ -299,6 +301,59 @@ impl SignedTransaction {
             }
         }
     }
+
+    pub fn validate_gas_limit(&self) -> Result<()> {
+        // Verify minimum gas requirement
+        match self {
+            SignedTransaction::Legacy { .. }
+            | SignedTransaction::Eip2930 { .. }
+            | SignedTransaction::Eip1559 { .. } => {
+                const MINIMUM_ETH_GAS: EvmGas = EvmGas(21_000);
```

Comment:
these constants (in fact, I would say, all constants) could do with being moved to a constants.rs, against the day when we trip over them later..

Reply:
https://github.com/Zilliqa/zq2/issues/1089